Documentation/install/get-dotnetcore-dnx-windows.md

```diff
@@ -25,35 +25,35 @@ You can see the currently installed DNX versions with `dnvm list`, which will di
 Installing a .NET Core DNX
 ==========================
 
-It's easy to install the latest .NET Core-based DNX, using the `dnvm install` command.
+It's easy to install the latest .NET Core-based DNX, using the `dnvm install` command. The `-u` (or `-Unstable`) parameter installs latest unstable version.
 
 	C:\coreclr-demo> dnvm install -r coreclr latest -u
 
-This will install the 32-bit version of .NET Core. If you want the 64-bit version, you can specify processor architecture.
+This will install the 32-bit version of .NET Core. If you want the 64-bit version, you can specify processor architecture:
 
 	C:\coreclr-demo> dnvm install -r coreclr -arch x64 latest -u
 
```

Comment:
It can be changed to:

\`\`\`cmd
C:\coreclr-demo> dnvm install -r coreclr -arch x64 latest -u 
\`\`\`

So it looks like:

```cmd
C:\coreclr-demo> dnvm install -r coreclr -arch x64 latest -u 
```